chinese-technical-terms.md

```diff
@@ -88,10 +88,10 @@ perceptron | 感知器 | 感知器 | gǎnzhī qì
 multi-layer perceptron | 多层感知器 | 多層感知器 | duōcéng gǎnzhī qì
 word embedding | 单词嵌入 | 單詞嵌入 | dāncí qiànrù
 convolutional (neural) network | 卷积(神经)网络 | 卷積(神經)網絡 | juàn jī (shénjīng) wǎngluò
-feature map |
-convolutional filter |
-stride |
-(max/min) pooling |
+feature map | 特征图 | 特徵圖 | tèzhēng tú
+convolutional filter | 卷积滤波器 | 卷積過濾器 | juàn jī guòlǜ qì
+stride | 步长 ｜ 步長 | bù zhǎng
```

Comment:
I think it should be ( bù cháng)?

btw, notice the `|` (en keybord) and `|` (zh keybord) are different. make sure you are typing '|' with English keybord layout, other wise there will be error in format.